src/pages/dashboard/Variables/VariableManagerContext.tsx

```
@@ -0,0 +1,446 @@
+        oldMeta.cleanup();
+      }
+
+      // 自动分析依赖关系
+      let dependencies: string[] = [];
+      const dependencySet = new Set<string>();
+      const validVarNames = new Set(variablesRef.current.map((v) => v.name));
+
+      if (variable.type === 'query') {
+        // 分析 definition 中的依赖
+        if (variable.definition) {
+          extractDependencies(variable.definition, validVarNames).forEach((dep) => dependencySet.add(dep));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)